recipes/pyqt/build.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+if [ `uname` == Linux ]; then
+    export QMAKESPEC="linux-g++"
+
+    # Add qt.conf to the right place in $SRC_DIR so that
+    # configure.py can run correctly
+    cp $PREFIX/bin/qt.conf $SRC_DIR
+fi
+
+if [ `uname` == Darwin ]; then
+    export DYLD_FALLBACK_LIBRARY_PATH=$PREFIX/lib/
+
+    # Add qt.conf to the right place in $SRC_DIR so that
+    # configure.py can run correctly
+    QTCONF_PLACE=$SRC_DIR/qtdirs.app/Contents/Resources
+    mkdir -p $QTCONF_PLACE
+    cp $PREFIX/bin/qt.conf $QTCONF_PLACE
+fi
+
+$PYTHON configure.py \
+          --verbose \
+          --confirm-license \
+          --bindir=$PREFIX/bin \
+          --destdir=$SP_DIR \
+          --qmake=$PREFIX/bin/qmake
+
+make
+make install
```

Comment:
We can verify that on the feedstock. I have very little time for this now ;-)

In any case, a much better test is the one in `run_test.py`, which verifies that the a gui application is started correctly.